README.md

```diff
@@ -1,204 +1,21 @@
 # Alfresco Identity Service
 
-The *Alfresco Identity Service* will become the central component responsible for identity-related capabilities needed by other Alfresco software, such as managing users, groups, roles, profiles, and authentication. Currently it deals just with authentication. This project contains the open-source core of this service.
+The Identity Service allows you to configure user synchronization between a supported LDAP provider or SAML identity provider and the Identity Service for Single Sign On (SSO) capabilities in other Alfresco software.
```

Comment:
The use of 'synchronization' here is slightly misleading.  I'd change that to 'authentication'.

Reply:
Updated to 'authentication' and made a similar alteration to the docs site.